.github/semantic.yml

```diff
@@ -0,0 +1,17 @@
+# https://github.com/Ezard/semantic-prs
+enabled: true
+titleAndCommits: true
+allowMergeCommits: false
+types:
+  - feat
+  - fix
+  - docs
+  - style
+  - refactor
+  - perf
+  - test
+  - build
+  - ci
+  - chore
```

Comment:
What's the difference between style, refactor and chore? And what about build vs. ci? The fewer keywords we use, the bigger chance we use them right.

Reply:
![image](https://github.com/TimefoldAI/timefold-solver/assets/247403/4e6a1899-882c-47bf-8a14-f3e684308393)

https://gist.github.com/brianclements/841ea7bffdb01346392c

Reply:
Thanks; I like "refactor" more than "chore", but in general, the categories are clear now.